jablib/src/main/java/org/jabref/logic/cleanup/FieldFormatterCleanupActions.java

```diff
@@ -151,30 +117,7 @@ private List<FieldChange> applyAllActions(BibEntry entry) {
     }
 
     public static List<FieldFormatterCleanup> parse(String formatterString) {
-        if ((formatterString == null) || formatterString.isEmpty()) {
```

Comment:
The method is now a one-liner - maybe remove competely (if I read your issue correctly)